systers_portal/templates/meetup/members.html

```diff
@@ -0,0 +1,22 @@
+{% extends "meetup/base.html" %}
+
+{% block meetup_location_page_content %}
+<div class="box-container">
+  <h3>Organizers</h3>
+  <ul>
+   {% for organizer in object.organizers.all %}       
+      <li>
+        <a href="{{ organizer.get_absolute_url }}">{{ organizer }}</a> 
```

Comment:
Also since we will be extracting the image urls maybe, it's worth doing [`select_related`](https://docs.djangoproject.com/en/1.8/ref/models/querysets/#select-related) for not hitting the database many-many times. Same for members.